src/main.c

```diff
@@ -1,79 +1,206 @@
-/// \file
```

Comment:
The existing example should remain in tiva cmake. Your code experiments should be separate.